operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainStatus.java

```diff
@@ -51,7 +52,7 @@
   @Description("Status of WebLogic Servers in this domain.")
   @Valid
   // sorted list of ServerStatus
-  List<ServerStatus> servers = new ArrayList<>();
+  private final List<ServerStatus> servers;
```

Comment:
should we make clusters private too?

Reply:
Shoot. I thought I had :)

Yeah, make it a TBD.